sklearn/datasets/_base.py

```diff
@@ -445,9 +447,8 @@ def load_descr(descr_file_name, *, descr_module=DESCR_MODULE):
     fdescr : str
         Content of `descr_file_name`.
     """
-    fdescr = _read_text(descr_module, descr_file_name)
-
-    return fdescr
+    path = resources.files(descr_module) / descr_file_name
+    return path.read_text()
```

Comment:
Same comment here. Let's pass:

```suggestion
    return path.read_text(encoding=encoding)
```

here and add `encoding="utf-8"` to `load_descr`.